.github/workflows/tests-integration-reusable.yml

```diff
@@ -30,7 +29,7 @@ jobs:
       run:
         shell: bash
     runs-on: ${{ inputs.os }} 
-
+    container: us-west2-docker.pkg.dev/shepherd-268822/shepherd2/concourse-resource:latest
```

Comment:
This seems to be a Google package repository. Can we rely on the stability of this URL, specifically the `268822`?

Reply:
This is the work around for now to use shepherd envs in workflows